src/client/components/LoginForm/LoginForm.css

```diff
@@ -0,0 +1,96 @@
+.container{
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+       
+}
+.heading-login{
+    width: 500px;
+}
+.button-image{
+    display: flex;
+    justify-content:space-around;
+}
+
+.button-image button{
+    width : 400px
+}
+
+p {
+    color: red;
+   }
+h2 {
+    overflow: hidden;
+    text-align: center;
+    }
+h2:before,
+h2:after {
+    background-color: rgb(223, 215, 215);
+    content: "";
+    display: inline-block;
+    height: 1px;
+    position: relative;
+    vertical-align: middle;
+    width: 50%;
+    }
+h2:before {
+    right: 0.5em;
+    margin-left: -50%;
+    }
+h2:after {
+    left: 0.5em;
+    margin-right: -50%;
+    }
+.github-logo{
```

Comment:
It is overwriting another class with the same name, so maybe make it more specific or change the name.